### PR TITLE
Added import storage docs

### PR DESCRIPTION
**Pull request by renebrandel**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/cli/storage/import.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* enable Lambda functions to access the chosen S3 or DynamoDB resource if you permit it
+
+This feature is particularly useful if you're trying to:
+* enable Amplify categories (such as API and function) to access your existing storage resources;
+* incrementally adopt Amplify for your application stack;
+* independently manage S3 and DynamoDB resources while working with Amplify.
+
+## Import an existing S3 bucket
+
+Select the "Content (Images, audio, video, etc.)" option when you've run `amplify import storage`. In order to successfully import your S3 bucket, your S3 bucket must be hosted in the same region as your Amplify project.
```
Comment (yuth) [inline docs/cli/storage/import.md:26]: We dont filter buckets based on region. One can have cross region buckets as those are considered global resources.
```suggestion
Select the "Content (Images, audio, video, etc.)" option when you've run `amplify import storage`.
```

File: docs/cli/storage/import.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Run `amplify push` to complete the import procedure.
+
+> Amplify projects are limited to one exactly S3 bucket.
+
+### Working with an imported S3 bucket with Amplify Libraries
+
+By default, Amplify Libraries assumes that S3 buckets are configured with the following folder access:
+- `public/` - TODO
```
Comment (yuth) [inline docs/cli/storage/import.md:35]: Will this be added before publishing of this PR?

**Review by yuth — CHANGES_REQUESTED** [review]
I am assuming those todos will be done before we merge

File: docs/cli/storage/import.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
+---
+title: Use an existing S3 bucket or DynamoDB table
+description: Configure the Amplify CLI to use existing S3 bucket or DynamoDB table resources as a storage resource for other Amplify categories. (API, function, and more)
```
Comment (attilah) [inline docs/cli/storage/import.md:3]: Function

**Review by attilah — APPROVED** [review]
Beside missing 'uploads/' policies it is looking good!

File: docs/cli/storage/import.md (hunk 1 above)
Comment (renebrandel) [reply docs/cli/storage/import.md]: Addressed the issue.

**Review by yuth — APPROVED** [review]

File: docs/cli/auth/import.md [hunk 4]
```diff
@@ -5,7 +5,11 @@ description: Configure the Amplify CLI to use existing Amazon Cognito User Pool
 
 Import existing Amazon Cognito resources into your Amplify project. Get started by running `amplify import auth` command to search for & import an existing Cognito User Pool & Identity Pool in your account.
```
Comment (jamesonwilliams) [inline docs/cli/auth/import.md:6]:
```suggestion
Import existing Amazon Cognito resources into your Amplify project. Get started by running the `amplify import auth` command to find and import Cognito resources that already exist in your account:
```

File: docs/cli/storage/import.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
+---
+title: Use an existing S3 bucket or DynamoDB table
+description: Configure the Amplify CLI to use existing S3 bucket or DynamoDB table resources as a storage resource for other Amplify categories. (API, Function, and more)
+---
+
+Import an existing S3 bucket or DynamoDB tables into your Amplify project. Get started by running `amplify import storage` command to search for & import an S3 or DynamoDB resource from your account.
```
Comment (jamesonwilliams) [inline docs/cli/storage/import.md:6]:
```suggestion
Import existing S3 buckets or DynamoDB tables into your Amplify project. Get started by running the `amplify import storage` command to find and import S3 or DynamoDB resources that already exist in your account. 
```

File: docs/cli/storage/import.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+Select the "S3 bucket - Content (Images, audio, video, etc.)" option when you've run `amplify import storage`. 
+
+Run `amplify push` to complete the import procedure.
+
+> Amplify projects are limited to exactly one S3 bucket.
+
+### Connect to an imported S3 bucket with Amplify Libraries
+
+By default, Amplify Libraries assumes that S3 buckets are configured with the following access patterns:
```
Comment (jamesonwilliams) [inline docs/cli/storage/import.md:34]:
```suggestion
By default, the Amplify libraries assume that S3 buckets are configured with the following access patterns:
```

File: docs/cli/storage/import.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+- `public/` - Accessible by all users of your app
```
Comment (jamesonwilliams) [inline docs/cli/storage/import.md:35]: Some of the libraries use `guest` here instead of `public`, right?

File: docs/cli/storage/import.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+- `protected/{user_identity_id}/` - Readable by all users, but writable only by the creating user 
+- `private/{user_identity_id}/` - Only accessible for the individual user
+
+You can either configure your IAM role to use the Amplify-recommended policies or in your Amplify libraries configuration [overwrite the default storage path behavior](~/lib/storage/configureaccess.md/q/platform/js#customize-object-key-path).
```
Comment (jamesonwilliams) [inline docs/cli/storage/import.md:39]:
```suggestion
You can either configure your IAM role to use the Amplify-recommended policies or you can [overwrite the default storage path behavior in your Amplify library configuration](~/lib/storage/configureaccess.md/q/platform/js#customize-object-key-path).
```

**Pull request merged**